win32ss/user/user32/controls/listbox.c

```diff
@@ -3041,9 +3041,11 @@ LRESULT WINAPI ListBoxWndProc_common( HWND hwnd, UINT msg,
         return LB_OKAY;
 
     case LB_GETLISTBOXINFO:
+#ifndef __REACTOS__
         if (descr->style & LBS_MULTICOLUMN) //// ReactOS
```

Comment:
What I see here is code inside `#ifndef __REACTOS__` block have `//// ReactOS` comment, this looks quite strange, where this code block comes from? I can't find it in Wine repository.

Reply:
@binarymaster 
    case LB_GETLISTBOXINFO:
        if (descr->style & LBS_MULTICOLUMN) //// ReactOS
           return descr->page_size * descr->column_width;
        else
           return descr->page_size;
 
This is the original code in win32ss/user/user32/controls/listbox.c,,  but in fact as I tested  on Windows, LB_GETLISTBOXINFO always returns the number of items in a column, whether MULTICOLUMN type or not.

Reply:
So... why `#ifndef __REACTOS__` ?

Reply:
<return descr-> page_size * descr-> column_width> (Wrong value)
only returns when __REACTOS__ is not defined. Correct?

Reply:
@binarymaster  Now is equal to WINE code